dhcp_options.go

```diff
@@ -91,26 +91,43 @@ func (odbi *ovndb) dhcpOptionsAddImp(cidr string, options map[string]string, ext
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
-func (odbi *ovndb) dhcpOptionsSetImp(cidr string, options map[string]string, external_ids map[string]string) (*OvnCommand, error) {
-
-	row, err := newDHCPRow(cidr, nil, external_ids)
-	if err != nil {
-		return nil, err
+func (odbi *ovndb) dhcpOptionsSetImp(uuid string, options map[string]string, external_ids map[string]string) (*OvnCommand, error) {
+	row := make(OVNRow)
+	dhcpUUID := ""
+	dhcpuuidlist,err := odbi.DHCPOptionsList()
+	if err != nil{
+		return nil, ErrorNotFound
 	}
-
-	dhcpUUID := odbi.getRowUUID(tableDHCPOptions, row)
-	if len(dhcpUUID) == 0 {
+	for _ ,v := range dhcpuuidlist{
+		if v.UUID == uuid{
+			dhcpUUID = uuid
+			break
+		}
+	}
+	if dhcpUUID == ""{
 		return nil, ErrorNotFound
 	}
+	if options != nil {
```

Comment:
may be if options is nil - return error?

Reply:
maybe we can make options to "" to clean all options.....

Reply:
done